pkg/config/polling_manager.go

```diff
@@ -205,7 +186,41 @@ func NewPollingProjectConfigManager(sdkKey string, pollingMangerOptions ...Optio
 	}
 
 	initDatafile := pollingProjectConfigManager.initDatafile
-	pollingProjectConfigManager.SyncConfig(initDatafile) // initial poll
+	if len(initDatafile) == 0 {
+		pollingProjectConfigManager.notificationCenter = registry.GetNotificationCenter(sdkKey)
```

Comment:
we initialize notification center in "else" so I think it can be done before if statement

Reply:
both initialisations serve different purposes, the one in `if` condition is before the `SyncConfig` call which makes sure notification is sent, the one in `else` is after the config has been set in which case notification won't be sent.

Reply:
cm.setConfig function should not send notification

Reply:
then where notification should be sent? I think setConfig has direct dependency of notification that's why it's sending notification in there.

Reply:
notification can be sent in SysConfig() when setConfig returns nil error